nion/swift/model/FileStorageSystem.py

```diff
@@ -527,7 +527,7 @@ def read_project_properties(self) -> typing.Dict:
                 properties_copy.setdefault("data_items", list()).append(data_item_properties)
 
         def data_item_created(data_item_properties: typing.Mapping) -> str:
-            earliest_datetime = datetime.datetime.fromtimestamp(time.mktime(time.gmtime(0))).isoformat()
+            earliest_datetime = datetime.datetime.fromtimestamp(time.mktime(time.localtime(0))).isoformat()
```

Comment:
Can you try `earliest_datetime = datetime.datetime.utcfromtimestamp(time.mktime(time.gmtime(0))).isoformat()` instead? All times are stored as GMT so I think this will be a cleaner solution as long as it works.

Reply:
This also doesn't work because the error happens in `time.mktime`, so it doesn't matter which function from the `datetime` module we're using.

Reply:
The actual problem is that on timezones with a "positive" offset (i.e. UTC+01, like central europe), `time.mktime(time.gmtime(0))` would be a negative number, which is not supported internally, hence the `OverflowError`.

Reply:
Sorry I missed these comments - what about just `earliest_datetime = datetime.datetime.utcfromtimestamp(0).isoformat()`

Reply:
My 2nd suggestion doesn't work. I've found a way to reproduce this error. I'll fix it separately.